developer/src/kmc-keyboard/test/fixtures/sections/layr/invalid-invalid-form.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!DOCTYPE keyboard SYSTEM "../../../../../../../resources/standards-data/ldml-keyboards/techpreview/dtd/ldmlKeyboard.dtd">
+<keyboard locale="und" conformsTo="techpreview">
+  <names>
+    <name value="layr-invalid-form" />
+  </names>
+
+  <keys>
+    <key id="grave" to="`" />
+    <key id="one" to="1" />
+  </keys>
+
+  <layers form="holographic">
```

Comment:
Can we add a comment about what is invalid here for future maintainers?

Reply:
done